test/integration/smart_answer_flows/experimental_flow_test_helper.rb

```diff
@@ -0,0 +1,231 @@
+module ExperimentalFlowTestHelper
+  def self.included(object)
+    object.extend(ClassMethods)
```

Comment:
Yup good point, I consider this commit's code to be a quick prototype and not example of a mergable commit. I plan to squash it away before an actual merge, it's mostly to serve as an example of the shoulda flow in case anyone wanted to try it out.